src/components/v2/SuccessfulTransactionMessage/index.tsx

```diff
@@ -59,3 +60,17 @@ export const SuccessfulTransactionMessage: React.FC<ISuccessfulTransactionMessag
     </div>
   );
 };
+
+export const SuccessfulTransactionModal: React.FC<
+  Omit<IModalProps, 'children'> & ISuccessfulTransactionMessageProps
```

Comment:
Small thing: it would be nice to export this type

Reply:
done